mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/WorkerExecutionOperationsNetworkStage.java

```diff
@@ -479,7 +484,7 @@ private void setupSubscriptionStateHandler(ExecuteStageRequest executeStageReque
     }
 
     private String getWorkerStringPrefix(int stageNum, int index, int number) {
-        return "stage " + stageNum + " worker index=" + index + " number=" + number;
+        return String.format("stage=%s worker index=%s number=%s", stageNum, index, number);
```

Comment:
the format is being changed here,
original would render `getWorkerStringPrefix(1,1,1)`
as `stage 1 worker index=1 number=1`
new would be `stage=1 worker index=1 number=1`

I hope nobody is parsing this to determine worker properties...

Reply:
Good catch!